domains/antineelta.json

```
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "yuyutsu07",
+           "email": "yuyutsurao7@gmail.com",
+           "discord": "916930338649939969"
+        },
+    
+        "record": {
+            "CNAME": "https://yutudrive.onrender.com/"
```

```suggestion
            "CNAME": "yutudrive.onrender.com"
```
Also this seems to be a knockoff version of google drive so this could be an abuse risk